quarkus/container/Dockerfile

```diff
@@ -18,11 +18,13 @@ RUN mv /tmp/keycloak/keycloak-* /opt/keycloak && mkdir -p /opt/keycloak/data
 RUN chmod -R g+rwX /opt/keycloak
 
 FROM registry.access.redhat.com/ubi8-minimal
+ENV LANG en_US.UTF-8
 
 COPY --from=build-env --chown=1000:0 /opt/keycloak /opt/keycloak
 
 RUN microdnf update -y && \
-    microdnf install -y java-11-openjdk-headless && microdnf clean all && rm -rf /var/cache/yum/* && \
+    microdnf install -y --nodocs java-11-openjdk-headless glibc-langpack-en && microdnf clean all && rm -rf /var/cache/yum/* && \
+    sed -i 's/^LANG=.*/LANG="en_US.utf8"/' /etc/locale.conf && \
```

Comment:
Is the `sed` step needed? Looking at the old container it doesn't do this:
https://github.com/keycloak/keycloak-containers/blob/main/server/Dockerfile#L12

Reply:
it is not strictly needed, tried it locally without and had same results, see first paragraph of the description of this PR. but it is stated to do it like this in the mentioned rh article so i left it there to reduce risk of side effects.

Reply:
also here: whatever works for you, i'm fine with it, just tell me and I adjust the PR

Reply:
If we don't believe it's required I'd prefer to remove it as that makes this PR consistent to how we fix this issue in the old container, which I think it's the safest path.